### PR TITLE
Feature html type

### PR DESCRIPTION
**Pull request by danield-ap**

### CODE REVIEW

File: src/Nox.Types/Types/Html/Html.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Html"/> type and value object.
+/// </summary>
+public sealed class Html : ValueObject<string, Html>
+{
+    public new static Html From(string value)
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types/Types/Html/Html.cs:10]: XML documentation missed for the public methods

File: src/Nox.Types/Types/Html/Html.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    /// <returns>true if the <see cref="Html"/> value is valid .</returns>
+    internal override ValidationResult Validate()
+    {
+        bool ContainsTag(string value, string tagName)
+        {
+            return Regex.IsMatch(value, $@"<{tagName}\s*>", RegexOptions.IgnoreCase) // opening tag
+                && Regex.IsMatch(value, $@"</{tagName}\s*>", RegexOptions.IgnoreCase); // closing tag
+        }
+
+        var result = base.Validate();
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types/Types/Html/Html.cs:39]: Can we have base.Validate() before these Regex validations.

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]
Promote it to a PR, I just have small comments that you can resolve.

**Review by rochar — APPROVED** [review]
Looks simple and good

I have question regarding to have or not TypeOptions.... but for now lets keep this way

File: src/Nox.Types.EntityFramework/Types/Html/HtmlConverter.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class HtmlConverter : ValueConverter<Html, string>
+{
+    public HtmlConverter() : base(uri => uri.Value, uriValue => Html.From(uriValue)) { }
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/Html/HtmlConverter.cs:7]: @danield-ap change the variable names

Comment (danield-ap) [reply src/Nox.Types.EntityFramework/Types/Html/HtmlConverter.cs]: Variable updated.

**Pull request merged**